### PR TITLE
PAGOPA-111 create api to get stations and channels csv

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Two APIs have been added: one to get all channels inside a csv file, one to get all stations inside a csv file.

#### How Has This Been Tested?
This addition has been tested via unit test in h2 environment

#### Screenshots (if appropriate):
Stations csv: 
Screenshot 2022-12-05 alle 11 14 54

Channels csv:
Screenshot 2022-12-05 alle 11 14 29

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FedericoRuzzier** [comment]
Csv canali:
Screenshot 2022-12-02 alle 17 35 23

**Comment by FedericoRuzzier** [comment]
Screenshots of the csv files are in the pr description

File: src/main/java/it/pagopa/pagopa/apiconfig/service/ChannelsService.java [hunk 1]
```diff
@@ -209,6 +223,30 @@ private List<List<String>> mapPspToCsv(List<ChannelPsp> pspList) {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * @param channelList list of all channels
+     * @return list of list of strings representing the CSV file
+     */
+    private List<List<String>> mapChannelsToCsv(List<Canali> channelList) { //ERRORE, HEADERS IN METODO CSV ESISTENTE SBAGLIATI
+        return channelList.stream()
+                .map(elem -> Arrays.asList(deNull(elem.getFkIntermediarioPsp().getCodiceIntermediario()),
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/pagopa/apiconfig/service/ChannelsService.java:232]: Can `getFkIntermediarioPsp()` return null?

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

File: src/main/java/it/pagopa/pagopa/apiconfig/service/StationsService.java [hunk 2]
```diff
@@ -159,6 +180,19 @@ private List<String> getCsvValues(StationCreditorInstitution elem) {
         return list;
     }
 
+    private List<String> getCsvValuesWithUrl(StationCreditorInstitution elem) {
+        List<String> list = getCsvValues(elem);
+        list.add("https://config" + getEnvironment() + ".platform.pagopa.it" + deNull(elem.getCreditorInstitutionCode()));
+        return list;
+    }
+
+    private String getEnvironment() {
```
[the remark this answers is not in the document]
Comment (FedericoRuzzier) [reply src/main/java/it/pagopa/pagopa/apiconfig/service/StationsService.java:189]: CommonUtil makes all elements inside static, they cannot access `@Value` annotations. A workaround is to change it to `@Component` and autowiring it in the needed classes.

[the remark this answers is not in the document]
Comment (jacopocarlini) [reply src/main/java/it/pagopa/pagopa/apiconfig/service/StationsService.java:189]: You can refactor the method in this way and pass the `@Value` from the components:
```suggestion
    private String getEnvironment(String env) {
```

[the remark this answers is not in the document]
Comment (FedericoRuzzier) [reply src/main/java/it/pagopa/pagopa/apiconfig/service/StationsService.java]: Done 👍

**Review by jacopocarlini — APPROVED** [review]
Great 💪

**Pull request merged**